README.md

```diff
@@ -191,6 +191,46 @@ public void ConfigureServices(IServiceCollection services)
 
 ```
 
+## Directory Client
+A new Directory Client can be creating as follows:
+```csharp
+
+   var logggerFactory = new NullLoggerFactory();
+   var options = new AsertoDirectoryOptions();
+   options.DirectoryServiceUrl = "url_and_port_to_directory_service";
+   options.DirectoryApiKey = "directory_api_key";
+   options.TenantID = "directory_tenant_id";
+   options.Insecure = false;
+   var optionsInt = Microsoft.Extensions.Options.Options.Create(options);
+   var directoryClient = new (optionsInt, logggerFactory);
+
+```
+you'll need to provide the directory service URL, an API key and the Tenant ID.
+The client can be configure to not use de https by providing `options.Insecure = true;`.
+
+Example call to the directory client:
+```csharp
+
+   public async Task GetObject()
+   {
+      //...
+
+      var directoryClient = new (optionsInt, logggerFactory);
+
+      var objIdentifier = new ObjectIdentifier();
+      objIdentifier.Type = "user";
+      objIdentifier.Key = "userID"; // user id of the user object you want to retrieve
+      var getObjectReq = new GetObjectRequest();
+      getObjectReq.Param = objIdentifier;
+
+      var getObjectResp = await directoryClient.GetObjectAsync(getObjectReq);
+
+      //...
+   }
```

Comment:
Could we provide a friendlier way to call the client, afaik one important thing for the SDK is to provide a cleaner interface than using the bindings directly, something like:
```csharp

var directoryClient = new (optionsInt, logggerFactory);

var getObjectResp = await directoryClient.GetObjectAsync({Type: "user", Key:  "userID"});
```